srcpkgs/libsecret/template

```diff
@@ -1,24 +1,28 @@
 # Template file for 'libsecret'
 pkgname=libsecret
-version=0.20.3
+version=0.20.4
 revision=1
 build_style=gnu-configure
 build_helper="gir"
 configure_args="$(vopt_enable gir introspection) --disable-static
  --with-libgcrypt-prefix=${XBPS_CROSS_BASE}/usr $(vopt_enable gir vala)"
-hostmakedepends="glib-devel gtk-doc pkg-config libxslt $(vopt_if gir vala)"
-makedepends="libgcrypt-devel $(vopt_if gir vala) libglib-devel"
+hostmakedepends="gtk-doc pkg-config libxslt $(vopt_if gir vala)"
+makedepends="glib-devel libgcrypt-devel $(vopt_if gir vala)"
 short_desc="GObject based library for accessing the Secret Service API"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1-or-later"
 homepage="http://www.gnome.org"
 distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
-checksum=4fcb3c56f8ac4ab9c75b66901fb0104ec7f22aa9a012315a14c0d6dffa5290e4
+checksum=325a4c54db320c406711bf2b55e5cb5b6c29823426aa82596a907595abb39d28
 
 # Package build options
 build_options="gir"
 build_options_default="gir"
 
+if [ "$CROSS_BUILD" ]; then
```

Comment:
There's not a compelling reason to guard this in a `$CROSS_BUILD` test; just add `glib-devel` to `hostmakedepends` and `makedepends` in all cases.